packages/ovm-toolchain/buidler.config.ts

```diff
@@ -23,9 +36,8 @@ const config: any = {
     timeout: 50000,
   },
   solc: {
-    path: path.resolve(__dirname, '../../node_modules/@eth-optimism/solc'),
+    version: '0.5.16',
```

Comment:
What does this change do?

Reply:
Made it so that it uses the normal compiler by default, doesn't use the OVM unless the --ovm flag is set.

Reply:
Ahh ok makes sense, cool!